engine/lite/embeddedDeploymentManager/src/main/scala/pl/touk/nussknacker/engine/embedded/StreamingDeploymentStrategy.scala

```diff
@@ -0,0 +1,67 @@
+package pl.touk.nussknacker.engine.embedded
```

Comment:
Can it be in `streaming` subpackage to separate streaming related things from core embedded mechanism?

Reply:
:+1: